app/controllers/api_controller/tags.rb

```diff
@@ -79,7 +79,7 @@ def edit_resource_tags(type, id, data)
     def delete_resource_tags(_type, id, _data = {})
       destroy_tag_and_classification(id)
       action_result(true, "tags id: #{id} deleting")
-    rescue => err
+    rescue BadRequestError => err
```

Comment:
I think we need here rescue any error except ActiveRecord::RecordNotFound somehow. BadRequestError will likely never occur.